static/scripts/onboarding/onboarding.ts

```diff
@@ -148,23 +147,15 @@ async function setConfig() {
   try {
     toggleLoader("start");
     const pluginKit = Octokit.plugin(createOrUpdateTextFile);
-    const octokit = new pluginKit({ auth: githubPAT.value });
-    const { data: userInfo } = await octokit.rest.users.getByUsername({
-      username: orgName.value,
+    const octokit = new pluginKit({ auth: getSessionToken() });
+    const { data: appInstallations } = await octokit.request("GET /orgs/{org}/installations", {
+      org: orgName.value,
+      per_page: 100,
     });
-    if (userInfo.type === "Organization") {
-      const repositoryId = await getRepoID(octokit, orgName.value, REPO_NAME);
+    const repositoryId = await upsertRepoID(octokit, orgName.value, REPO_NAME);
+    const ins = appInstallations.installations.filter((installation) => installation.app_id === APP_ID);
```

Comment:
```suggestion
    const installations = appInstallations.installations.filter((installation) => installation.app_id === APP_ID);
```

You should never shorthand words. This is to reduce cognitive overhead for other developers.

Reply:
The only exceptions I can think of are "repo" and "org" because octokit uses that terminology

Reply:
Totally agree, although I am not the one who wrote these, changing now!